src/components/manage/Widgets/ArrayWidget.jsx

```diff
@@ -32,13 +32,17 @@ const CreatableSelect = loadable(() => import('react-select/creatable'));
 
 const messages = defineMessages({
   select: {
-    id: 'Select…',
-    defaultMessage: 'Select…',
+    id: 'Select...',
```

Comment:
I think we never used this consistently in Plone core. I was wondering if there is anybody we can ask how to use the "..." properly in translations...

Reply:
I vaguely remember @albertcasado in favor of using the “unified” single character for it.

Reply:
Then we might want to keep using the unified single character. If I am not mistaken this is used most of the time in the Plone translations anyways.

@giuliaghisini would you be ok with changing "..." to the unified character? Maybe someone steps up to go through the Volto core and replaces all occurrences of "...".

Reply:
ok, yes!